02Arrays/Array/Tester.swift

```diff
@@ -54,16 +104,64 @@ class Tester {
         Ex. [ I, I, O] -> 2
      */
     func minimumChairs(array: [Character]) -> Int {
-        1
+        var count = 0
+        for i in 0..<array.count {
+            if array[i] == "I" && i != 0 {
+                var prev = i-1
```

Comment:
does not account for multiple "O" in a row, only look at i-1, not i-2 or i-3, etc.
Don't keep track of prev "O", but keep track of the max "I"